Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -59,16 +63,33 @@ let animationsReducer = Reducer<AnimationsState, AnimationsAction, AnimationsEnv
   state, action, environment in
 
   switch action {
+  case .alertButtonTapped:
+    state = .init()
+    return .none
+    
   case let .circleScaleToggleChanged(isScaled):
     state.isCircleScaled = isScaled
     return .none
+    
+  case .dismissAlert:
+    state.alert = nil
+    return .none
 
   case .rainbowButtonTapped:
     return .keyFrames(
       values: [Color.red, .blue, .green, .orange, .pink, .purple, .yellow, .white]
         .map { (output: .setColor($0), duration: 1) },
       scheduler: environment.mainQueue.animation(.linear)
     )
+    
+  case .resetButtonTapped:
+    state.alert = .init(
+      title: .init("Reset state?"),
+      primaryButton: .destructive(.init("Reset"))
+        .send(.alertButtonTapped, withAnimation: .default),
```

Comment:
Instead of this "fluent" style, what do you think of adding `animation` as a parameter to these button helpers?
```suggestion
      primaryButton: .destructive(.init("Reset"), animation: .default),
```

Reply:
Sounds good. Already considered this as an alternative.